src/utilities/Libify.js

```diff
@@ -459,6 +459,68 @@ Libify.Operation.revokeSponsorship = function(opts) {
   }
 }
 
+Libify.Operation.clawback = function(opts) {
+  assertNotEmpty(opts.asset, 'Clawback operation requires asset');
+  assertNotEmpty(opts.from, 'Clawback operation requires account from which the asset is clawed back');
+  assertNotEmpty(opts.amount, 'Clawback operation requires amount clawed back');
+  return Sdk.Operation.clawback({
+    asset: Libify.Asset(opts.asset),
+    from: opts.from,
+    amount: opts.amount,
+    source: opts.sourceAccount,
+  })
+}
+
+Libify.Operation.clawbackClaimableBalance = function(opts) {
+  assertNotEmpty(opts.balanceId, 'Clawback Claimable Balance operation requires claimable balance ID');
+
+  return Sdk.Operation.clawbackClaimableBalance({
+    balanceId: opts.balanceId,
+  });
```

Comment:
You can just pass `opts` directly, no?
```suggestion
  return Sdk.Operation.clawbackClaimableBalance(opts);
```

Reply:
Also, the validation is done by `clawbackClaimableBalance`, but I get it if you want to double-check the inputs.

Reply:
Good suggestion! `opts` are passed explicitly everywhere else, so trying to keep it consistent. We're validating on the frontend to make the UX better (if you're submitting a bunch of operations, it might not be great to get an error telling you to fix a simple thing like that).